proto/platform/registry/v1/registry.platform.proto

```diff
@@ -394,3 +406,10 @@ message DiffImageRequest {
   string from_digest = 2;
   string to_digest = 3;
 }
+
+message RepoDigestRequest {
```

Comment:
`RepoDigestRequest` and not `SbomRequest` because this is meant to be shared between GetSbom and GetVulnReport (forthcoming), lmk if it should be split!